### PR TITLE
Add .git-blame-ignore-revs file

### PR DESCRIPTION
**Pull request by m-holger**
Ignore major reformatting commits

### CODE REVIEW

**Review by jberkenbilt — APPROVED** [review]
Great start. If I find myself stumbling over earlier bulk changes, I can update as needed (or you can or anyone else). We can try to remember to keep this up to date or add a note in README-maintainer. I'll await your response before merging and will be offline for the rest of the day soon.

**Comment by m-holger** [comment]
I have updated the note in README-maintainer

**Review by jberkenbilt — APPROVED** [review]
Great.

**Pull request merged**
